File: A wolf in sheep's clothing/A wolf in sheep's clothing.js
function warnTheSheep(queue) {
  if(queue.indexOf("wolf")==queue.length-1)return "Pls go away and stop eating my sheep"
  return `Oi! Sheep number ${(queue.length-1)-queue.indexOf("wolf")}! You are about to be eaten by a wolf!`
}

File: A wolf in sheep's clothing/A wolf in sheep's clothing.py
def warn_the_sheep(queue):
    queue = queue[::-1]
    search_idx = queue.index("wolf")
    return 'Pls go away and stop eating my sheep' if search_idx==0 else "Oi! Sheep number {}! You are about to be eaten by a wolf!".format(search_idx)
